templates/tinymcenext/template.js

```diff
@@ -24,102 +23,102 @@ var namespaceDescriptions = {
  * @return {[type]}        [description]
  */
 exports.template = function (root, toPath) {
-	var archive = new ZipWriter();
-	var rootTemplate = compileTemplate('root.handlebars');
-	var namespaceTemplate = compileTemplate('namespace.handlebars');
-	var template = compileTemplate('member.handlebars');
-
-	// bind new archive to function
-	addPageToArchive = addPageToArchive.bind(archive); // jshint ignore:line
-
-	// sort types alphabetically
-	var sortedTypes = root.getTypes().sort(function (a, b) {
-		if (a.fullName < b.fullName) return -1;
-  	if (a.fullName > b.fullName) return 1;
-  	return 0;
-	});
-
-	addPageToArchive({
-		filename: '_data/nav_api.yml',
-		content: YAML.safeDump(getNavFile(sortedTypes))
-	});
-
-	// create all yml and md for each item
-	var pages = sortedTypes.map(getMemberPages.bind(null, root, template))
-	flatten(pages).forEach(addPageToArchive);
-
-	getNamespacesFromTypes(sortedTypes).map(function (namespace) {
-		var fileName = ('api/' + namespace + '/index.html').toLowerCase();
+  var archive = new ZipWriter();
+  var rootTemplate = compileTemplate('root.handlebars');
+  var namespaceTemplate = compileTemplate('namespace.handlebars');
+  var template = compileTemplate('member.handlebars');
+
+  // bind new archive to function
+  addPageToArchive = addPageToArchive.bind(archive); // jshint ignore:line
+
+  // sort types alphabetically
+  var sortedTypes = root.getTypes().sort(function (a, b) {
+    if (a.fullName < b.fullName) return -1;
+    if (a.fullName > b.fullName) return 1;
+    return 0;
+  });
+
+  addPageToArchive({
+    filename: '_data/nav_api.yml',
+    content: YAML.dump(getNavFile(sortedTypes))
```

Comment:
I assume this was needed due to the js-yaml 3 -> 4 conversion?

Reply:
https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump
> Older safe* functions were renamed.

They dropped the "safe" prefixes